### PR TITLE
Add short hand notation for billion

### PR DESCRIPTION
**Pull request by varnikaj**
Add short hand notation for billion

### CODE REVIEW

File: src/number-formatting/short-format.js [hunk 1]
```diff
@@ -40,6 +40,11 @@ function shortFormat(num, options) {
         }
     }
 
+    if (digitSize >= 10) { // > 999,999,500
```
Comment (michax) [inline src/number-formatting/short-format.js]: Nitpic. In comment example we have 9 digit number. Is that correct?

Comment (varnikaj) [reply src/number-formatting/short-format.js]: Yes, it should be 9 digit number only, so any 9 digit number after mentioned number is rounded off to billion

Comment (lukeapage) [reply src/number-formatting/short-format.js]: I'm not sure the comments make sense..
surely

if (digitSize >= 10) { // >= 1,000,000,000

File: test/unit/number-formatting/format-spec.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ describe('NumberFormatting format', () => {
             expect(shortFormat(1050000)).toEqual('1.05m');
             expect(shortFormat(10500000)).toEqual('10.5m');
             expect(shortFormat(105000000)).toEqual('105m');
-            expect(shortFormat(1050000000)).toEqual('1,050m');
+            expect(shortFormat(1050000000)).toEqual('1.05bn');
```
Comment (lukeapage) [inline test/unit/number-formatting/format-spec.js]: I think you should add more than a single assertion. E.g. the boundaries on both sides you expect. a larger number. etc.

File: short-format.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+// -- Exported methods section --
+
+/**
+ * Converts from a number to a string like "1k" or "100m".
+ * @param num
+ * @param precision
+ * @param options
+ * @returns {string} Returns 0 when dates are equal. -1 when date1 less than date2. 1 when date1 greater than date2.
+ */
```
Comment (varnikaj) [inline short-format.js]: Refactored code a bit

**Pull request merged**